File: ALGORITHMS/C++/min.cpp
#include <iostream>
using namespace std;


void MIN(int* tab, int size){

    int minVal = tab[0];
    for(int i =0; i<size;i++){
        if(tab[i]<minVal){
            minVal=tab[i];
        }
    }

    cout<<"MIN: "<<minVal<<endl;
}

void MAX(int* tab, int size){

    int maxVal = tab[0];
    for(int i =0; i<size;i++){
        if(tab[i]>maxVal){
            maxVal=tab[i];
        }
    }

    cout<<"MAX: "<<maxVal<<endl;
}

int main(){

    int tab[]={2,34,1,33333,3};
    int size = sizeof(tab)/sizeof(tab[0]);

    MIN(tab,size);
    MAX(tab,size);


}

File: ALGORITHMS/PYTHON/min.py
tab=[7,15,2,4,23]

size = len(tab)

def minVal(tab):
    minVal = tab[0]
    for i in tab:
        if i <minVal:
            minVal=i
    
    return minVal


def maxVal(tab):
    maxVal = tab[0]
    for i in tab:
        if i >maxVal:
            maxVal=i
    
    return maxVal


def avarageVal(tab):

    size = len(tab)

    i = 0
    suma = 0
    while i<size:
        suma+=tab[i]
        i+=i

    avarage = suma/size
    return avarage        


print(f"MIN: {minVal(tab)}\n")
print(f"MAX: {maxVal(tab)}\n")
print(f"AVARAGE: {avarageVal(tab)}\n")
